src-electron/util/env.js

```diff
@@ -31,10 +31,22 @@ export function defaultFileFormat() {
   return saveFileFormat
 }
 
+/**
+ *
+ * @returns path to zcl.json file
+ */
```

Comment:
Minor but for future references have all functions go into the following format:

> /**
 * Description:
 * Example:
 * @param {*} param1
 * @param {*} param2
 * @returns: returns ---
 */